tools/lambda-promtail/lambda-promtail/s3.go

```diff
@@ -135,6 +138,7 @@ func processS3Event(ctx context.Context, ev *events.S3Event) error {
 
 	}
 
+	fmt.Println("sending S3Event to promtail\n")
```

Comment:
`Println()` already adds a newline
```suggestion
	fmt.Println("sending S3Event to promtail")
```